core/coreunix/add.go

```diff
@@ -470,6 +472,9 @@ func (adder *Adder) addFile(file files.File) error {
 		return err
 	}
 
+	if !strings.EqualFold(adder.WpName, "") && adder.Wrap {
```

Comment:
This is the place a normal file is added, not the directory wrapper.

Reply:
i have rebase repo and please help me review this pr @schomatis

Reply:
thank you!

Reply:
Let's drop the `adder.Wrap` check.